scripts/licenses/licenses/library_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package licenses
+
+import (
+	"go/build"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
```

Comment:
We've normally used `pretty.Compare` in the past -- does this make life much easier?

Reply:
`cmp` makes it easy to compare slices when you don't care if the order of elements is different (using `cmopts.SortSlices()`). I'm not aware of a simple way to do the same using `pretty.Compare`.